### PR TITLE
[BAC-160] Parameterize curator options

### PR DESCRIPTION
**Pull request by alika**
**What this PR does / why we need it**: This PR makes the options to set how long to retain logs in the curator cronjob configurable from the chart values file.

**Which issue this PR fixes**:
This is part of [BAC-160][160]

**Verification Notes**:

To verify this you can install this chart into a dev cluster:
```
CHART_NAME=curator build/build.sh
helm install --name curator-0 --namespace curator-test --set schedule="00 2 * * *"  ./curator
kubectl -n curator-test describe cronjobs/curator
```
You should see the corresponding `--set` schedule value of `00 2 * * *`

[160]: https://samsung-cnct.atlassian.net/browse/BAC-160

### CODE REVIEW

**Comment by coffeepac** [comment]
would it be better to take the work in #13 that turns the entire config stanza into a values block?

File: README.md [hunk 1]
```diff
@@ -18,3 +18,17 @@ helm registry install quay.io/samsung_cnct/curator
...
+
+| Parameter                | Description                                     | Default                                 |
+| ------------------------ | ----------------------------------------------- | --------------------------------------- |
+| `name`                   | Name of the chart                               | `curator`                               |
+| `image.name`             | FQDN repository/image name                      | `quay.io/samsung_cnct/curator-container`|
+| `image.tag`              | Image tag                                       | `latest`                                |
+| `schedule`               | The cron schedule                               | `30 3 * * *`                            |
+| `retention.pattern`      | Prefix pattern of index to retain               | `logstash-`                             |
+| `retention.unit`         | Retention period unit                           | `days`                                  |
+| `retention.count`        | Retention period                                | `14`                                    |
```
Comment (guineveresaenger) [inline README.md]: I love this table. Could we add an example on how to change the default, like in your ToReview instructions, and perhaps a link to standard helm options? :D

**Comment by alika** [comment]
@coffeepac That is a much more flexible and reusable way to parameterize the options to the curator. I'll modify this PR as such.

**Comment by alika** [comment]
@coffeepac @guineveresaenger: I've updated the configuration values for the curator and the documentation. Let me know what you think.

File: README.md [hunk 2]
```diff
@@ -4,17 +4,120 @@
...
-[Curator github](https://github.com/elastic/curator)    
-[Curator Reference Documentation](https://www.elastic.co/guide/en/elasticsearch/client/curator/current/index.html)            
-[Discuss Curator](https://discuss.elastic.co/search?q=curator)
+[1]: https://github.com/app-registry/appr-helm-plugin/blob/master/README.md#install-the-helm-registry-plugin "install helm registry plugin"
+[2]: https://github.com/elastic/curator "curator github"
+[3]: https://www.elastic.co/guide/en/elasticsearch/client/curator/current/index.html "curator reference documentation"
+[4]: https://discuss.elastic.co/search?q=curator "curator discussion"
+[5]: https://www.elastic.co/guide/en/elasticsearch/client/curator/5.2/filters.html "curator filter documentation"
+[6]: https://www.elastic.co/guide/en/elasticsearch/client/curator/5.2/options.html "currator options documentation"
+[7]: https://www.elastic.co/guide/en/elasticsearch/client/curator/5.2/configfile.html "currator config file  documentation"
```
Comment (guineveresaenger) [inline README.md]: TIL this is a thing you can do. Makes review smoother!

**Comment by guineveresaenger** [comment]
Thank you for an awesome writeup on how to use it!

**Pull request merged**